cmd/metrics/cmd/mock_dest_store.go

```diff
@@ -69,7 +69,7 @@ func (mds *mockDestStore) Put(ctx context.Context, key string, source io.Reader,
 	return nil
 }
 
-func (mds *mockDestStore) Delete(ctx context.Context, key string) error {
+func (mds *mockDestStore) Delete(ctx context.Context, key string, opts ...storage.Option) error {
```

Comment:
Adding options to a Store should follow a different pattern.
Rather than cluttering the interface with `opts ...storage.Option`, this is made once in `New()`

Reply:
resolved at cfae952 , currently in `j2240-labelversions--wip-05-rev` after squash and rebase of this branch.

note: cfae952 passes `go_test` while `fuse_sidecar_test` fails, hence the rebase and try-again.